templates/tomcat/server.xml.erb

```diff
@@ -78,7 +78,7 @@
                SSLEnabled="true"
                maxThreads="150" scheme="https" secure="true"
                clientAuth="want"
-               sslProtocols="<%= scope['::candlepin::tls_versions'].map { |version| "TLSv#{version}"}.join(",") %>"
+               sslProtocol="<%= scope['::candlepin::tls_versions'].map { |version| "TLSv#{version}"}.join("\+") %>"
```

Comment:
```suggestion
               sslProtocol="<%= scope['::candlepin::tls_versions'].map { |version| "TLSv#{version}"}.join(",") %>"
```

The comma was correct all along for JSSE configuration.  Sorry for the confusion.